install-raspbian.sh

```diff
@@ -114,6 +114,24 @@ else
     tar -xzvf /tmp/photobooth-latest.tar.gz -C /var/www/html/
 fi
 
+if [ -f "/etc/nginx/sites-enabled/default" ]; then
```

Comment:
I think it should be okay from an Nginx perspective now. 👍 

One thing I personally would change, is to put the path to the nginx-config into a variable:

```bash
nginx_conf="/etc/nginx/sites-enabled/default"
if [ -f "${nginx_conf}" ]; then
  cp "${nginx_conf}" ...
  sed -i '...' "${nginx_conf}"
...
```
	

```

Reply:
Thanks, just shutdown my PC. I'll update tomorrow evening :)

Reply:
That's no big deal, I just don't like "code duplication" a lot (I know in my example I did it like that :p).

Reply:
```
diff --git a/install-raspbian.sh b/install-raspbian.sh
index ac4a69c..4086667 100755
--- a/install-raspbian.sh
+++ b/install-raspbian.sh
@@ -114,14 +114,16 @@ else
     tar -xzvf /tmp/photobooth-latest.tar.gz -C /var/www/html/
 fi
 
-if [ -f "/etc/nginx/sites-enabled/default" ]; then
+nginx_conf="/etc/nginx/sites-enabled/default"
+
+if [ -f "${nginx_conf}" ]; then
     info "### Enable PHP in NGINX"
-    cp /etc/nginx/sites-enabled/default ~/nginx-default.bak
-    sed -i 's/^\(\s*\)index index\.html\(.*\)/\1index index\.php index\.html\2/g' /etc/nginx/sites-available/default
-    sed -i '/location ~ \\.php$ {/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
-    sed -i '/include snippets\/fastcgi-php.conf/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
-    sed -i '/fastcgi_pass unix:\/run\/php\//s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
-    sed -i '/.*fastcgi_pass unix:\/run\/php\//,// { /}/s/^\(\s*\)#/\1/g; }' /etc/nginx/sites-available/default
+    cp "${nginx_conf}" ~/nginx-default.bak
+    sed -i 's/^\(\s*\)index index\.html\(.*\)/\1index index\.php index\.html\2/g' "${nginx_conf}"
+    sed -i '/location ~ \\.php$ {/s/^\(\s*\)#/\1/g' "${nginx_conf}"
+    sed -i '/include snippets\/fastcgi-php.conf/s/^\(\s*\)#/\1/g' "${nginx_conf}"
+    sed -i '/fastcgi_pass unix:\/run\/php\//s/^\(\s*\)#/\1/g' "${nginx_conf}"
+    sed -i '/.*fastcgi_pass unix:\/run\/php\//,// { /}/s/^\(\s*\)#/\1/g; }' "${nginx_conf}"
 
     info "### Testing NGINX config"
     /usr/sbin/nginx -t -c /etc/nginx/nginx.conf
@@ -129,7 +131,7 @@ if [ -f "/etc/nginx/sites-enabled/default" ]; then
     info "### Restarting NGINX"
     systemctl reload nginx
 else
-    error "Can not find /etc/nginx/sites-enabled/default"
+    error "Can not find ${nginx_conf}!"
 fi
 
 info "### Setting permissions."
```